torch/package/package_importer.py

```diff
@@ -583,6 +617,13 @@ def _add_extern(self, extern_name: str):
             return  # the shorter extern covers this extern case
         package.children[last] = _ExternNode()
 
+    def _add_selective_extern(self, selective_extern_name: str):
+        *prefix, last = selective_extern_name.split(".")
+        package = self._get_or_create_package(prefix)
+        if isinstance(package, _SelectiveExternNode):
```

Comment:
This case should be an assertion error. We cannot produce a package with a selective extern inside a selective extern, right?

Reply:
yeah that's a p gross logic error